tools/clang/lib/SPIRV/SpirvFunction.cpp

```diff
@@ -19,8 +19,8 @@ SpirvFunction::SpirvFunction(QualType returnType, SourceLocation loc,
                              llvm::StringRef name, bool isPrecise)
     : functionId(0), astReturnType(returnType), returnType(nullptr),
       fnType(nullptr), relaxedPrecision(false), precise(isPrecise),
-      containsAlias(false), rvalue(false), functionLoc(loc),
-      functionName(name) {}
+      containsAlias(false), rvalue(false), functionLoc(loc), functionName(name),
+      isWrapperOfEntry(false), debugScope(nullptr) {}
 
 SpirvFunction::~SpirvFunction() {
```

Comment:
You also need to release memory for the newly-added `debugScope` and `debugDeclares` I believe.

Reply:
done